doc/OWNER.md

```diff
@@ -0,0 +1,8 @@
+# WasmEdge Maintainers
+
+| Maintainer | GitHub ID | Company | Email |
+| --------------- | --------- | ----------- | ----------- |
+| Michael Yuan | @juntao | Second State | <michael@secondstate.io> |
+| Hydai Tai | @Hydai | Second State | <hydai0@secondstate.io> |
```

Comment:
Hung-Ying Tai | @hydai 
The email address is also wrong.

Reply:
Sorry for my mistake. fixed.